raiden-ts/CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 - [#1649] Have constant error messages and codes in public Raiden API.
 - [#1657] Expose RaidenChannel's id,settleTimeout,openBlock as required properties
 - [#1708] Expose RaidenTransfer's secret as optional property
+- [#1764] All transfers become monitored per default to make receiving transfers safe
```

Comment:
Please, whenever possible, prefer to link to the issue (which usually is more descriptive) on the changelog, instead of the PR. PRs can be linked if it happens to include quickfixes not described on any issue and which are user facing, but usually anything big enough should have its own issue linked as well, even if a single PR closes more than one issue.

Reply:
:heavy_check_mark: